src/main/java/org/traccar/model/Network.java

```diff
@@ -27,8 +27,10 @@ public class Network {
     public Network() {
     }
 
-    public Network(CellTower cellTower) {
-        addCellTower(cellTower);
+    public Network(CellTower... cellTowers) {
```

Comment:
If we're changing this, it probably makes sense to change the wifi constructor as well. For consistency

Reply:
oh good point